Cargo.toml

```diff
@@ -13,10 +16,14 @@ description = "a small crate to work with NEAR Gas unit values ergonomically and
 serde = { version = "1", features = ["derive"], optional = true }
 borsh = { version = "1", features = ["derive"], optional = true }
 schemars = { version = "0.8.8", optional = true }
-interactive-clap = { version = "0.2.4", optional = true }
+interactive-clap = { version = "0.3.0", optional = true }
```

Comment:
Let's use a range of versions here. We do not re-export it here, we just impl a trait from there.